packages/ai-constructs/API.md

```diff
@@ -1,120 +1,120 @@
-import { FunctionResources } from '@aws-amplify/plugin-types';
-import { ResourceProvider } from '@aws-amplify/plugin-types';
-import { ToolInputSchema } from '@aws-sdk/client-bedrock-runtime';
-import { ToolResultContentBlock } from '@aws-sdk/client-bedrock-runtime';
-
-declare namespace conversation {
-    export {
-        ConversationHandlerFunction,
-        ConversationHandlerFunctionProps,
-        runtime
-    }
-}
-export { conversation }
-
-// @public
-class ConversationHandlerFunction extends Construct implements ResourceProvider<FunctionResources> {
-    constructor(scope: Construct, id: string, props: ConversationHandlerFunctionProps);
-    // (undocumented)
-    resources: FunctionResources;
-}
-
-// @public (undocumented)
-type ConversationHandlerFunctionProps = {
-    entry?: string;
-    models: Array<{
-        modelId: string;
-        region?: string;
-    }>;
-};
-
-// @public (undocumented)
-type ConversationMessage = {
-    role: 'user' | 'assistant';
-    content: Array<ConversationMessageContentBlock>;
-};
-
-// @public (undocumented)
-type ConversationMessageContentBlock = {
-    text: string;
-};
-
-// @public (undocumented)
-type ConversationTurnEvent = {
-    conversationId: string;
-    currentMessageId: string;
-    responseMutation: {
-        name: string;
-        inputTypeName: string;
-        selectionSet: string;
-    };
-    graphqlApiEndpoint: string;
-    modelConfiguration: {
-        modelId: string;
-        systemPrompt: string;
-        region?: string;
-        inferenceConfiguration?: {
-            maxTokens?: number;
-            temperature?: number;
-            topP?: number;
-        };
-    };
-    request: {
-        headers: {
-            authorization: string;
-        };
-    };
-    messages: Array<ConversationMessage>;
-    toolsConfiguration?: {
-        dataTools?: Array<ToolDefinition & {
-            graphqlRequestInputDescriptor: {
-                queryName: string;
-                selectionSet: string[];
-                propertyTypes: Record<string, string>;
-            };
-        }>;
-        clientTools?: Array<ToolDefinition>;
-    };
-};
-
-// @public (undocumented)
-type ExecutableTool = ToolDefinition & {
-    execute: (input: DocumentType | undefined) => Promise<ToolResultContentBlock>;
-};
-
-// @public
-const handleConversationTurnEvent: (event: ConversationTurnEvent, props?: {
-    tools?: Array<ExecutableTool>;
-}) => Promise<void>;
-
-declare namespace runtime {
-    export {
-        ConversationMessage,
-        ConversationMessageContentBlock,
-        ConversationTurnEvent,
-        ExecutableTool,
-        handleConversationTurnEvent,
-        ToolDefinition
-    }
-}
-
-// @public (undocumented)
-type ToolDefinition = {
-    name: string;
-    description: string;
-    inputSchema: ToolInputSchema;
-};
-
-// (No @packageDocumentation comment for this package)
-
-```
+## API Report File for "@aws-amplify/ai-constructs"

```

Comment:
Can you check why this file shows large diff and regenerate this is such a way that it doesnt?

This might be happening if you have this configured https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings .

Reply:
Thanks for the reference. Addressed in [f242254](https://github.com/aws-amplify/amplify-backend/pull/1907/commits/f242254fbe96a419e96a8d5359eea8b24a934997)